Doc/distutils/apiref.rst

```diff
@@ -2039,7 +2039,7 @@ This is described in more detail in :pep:`301`.
 ===================================================================
 
 .. module:: distutils.command.check
-   :synopsis: Check the metadata of a package
+   :synopsis: Check the meta-data of a package
```

Comment:
I think this line was correct in the first place. Is there a reason to change `metadata` to `meta-data`?

Reply:
I did it for consistency, checkout the given line.

https://github.com/python/cpython/pull/17363/files#diff-007e8820cfbca8cecb1463f7c65c19b4L2038